src/enclave/enclave_oe.c

```diff
@@ -351,8 +394,12 @@ static void _copy_shared_memory(const sgxlkl_shared_memory_t* host)
         for (size_t i = 0; i < enc->num_virtio_blk_dev; i++)
         {
             enc->virtio_blk_dev_mem[i] = host->virtio_blk_dev_mem[i];
```

Comment:
As discussed in [issue 745](https://github.com/lsds/sgx-lkl/issues/745), a "shawdow" virtio_dev structure inside the enclave is needed.

Reply:
Yes, that will not be part of this PR. Someone with detailed knowledge of the virtio interface will have to work that out.